extractor/src/main/scala/com/digitalasset/extractor/writers/postgresql/Queries.scala

```diff
@@ -303,10 +303,8 @@ object Queries {
           )
         case V.ValueInt64(value) => Fragment("?", value)
         case V.ValueNumeric(value) =>
-          // FixMe: https://github.com/digital-asset/daml/issues/2289
-          //  For now all the ValueNumeric should have scale 10
-          //  Check this code once it is possible to create Numerics with different scale
-          Fragment("?::numeric(38,10)", value: BigDecimal)
+          assert(value.scale == 10)
```

Comment:
I guess we need to remove this assertion once we release daml-lf?

Reply:
In order to be able to merge this PR soon, I dropped the change for the extractor.
I will do in a other PR, next week.